prusa/link/web/cameras.py

```diff
@@ -35,8 +42,13 @@ def photo_by_camera_id(camera_id, req):
                             message=f"Camera with id: {camera_id} did not "
                                     f"take a photo yet.")
 
+    if trigger_scheme in (TriggerScheme.TEN_SEC,
+                          TriggerScheme.THIRTY_SEC,
+                          TriggerScheme.SIXTY_SEC):
+        photo_timeout = TRIGGER_SCHEME_TO_SECONDS[trigger_scheme]
+
```

Comment:
I would definitely use the dictionary keys instead of creating a new set to compare against. What if the dictionary contents change? This will break and nobody will have a clue. A step nicer would be to use the get method. It does everything you want neatly.
`photo_timeout = TRIGGER_SCHEME_TO_SECONDS.get(trigger_scheme, DEFAULT_PHOTO_TIMEOUT)`

Reply:
Good point